src/server/api/controllers/modules.controller.js

```diff
@@ -7,7 +7,7 @@ const moment = require('moment-timezone');
 
 const getModules = async () => {
   try {
-    return await knex('modules').select('modules.id', 'modules.title');
+    return await knex('modules').select('modules.id');
```

Comment:
Just id is not interesting - we need to get the module's name out as well.

Reply:
In the migration file there is no column called `title`. What should be done?